webpack/index.js

```diff
@@ -0,0 +1,10 @@
+import componentRegistry from 'foremanReact/components/componentRegistry';
+import { mount } from 'foremanReact/common/MountingService';
+import App from './containers/Application';
```

Comment:
App is probably a too generic name, that would mean no other plugin could use that name (nor core).

Reply:
I thought the name that mattered was the name you registered the component with. We're using `xui_katello` below.